MdeModulePkg/Bus/Pci/NonDiscoverablePciDeviceDxe/NonDiscoverablePciDeviceIo.c

```diff
@@ -1735,6 +1820,32 @@ STATIC CONST EFI_PCI_IO_PROTOCOL  PciIoTemplate =
   0
 };
 
+// MU_CHANGE [BEGIN]
+
+/**
+  Event notification that is fired when IOMMU protocol is installed.
+
+  @param  Event                 The Event that is being processed.
+  @param  Context               Event Context.
+
+**/
+VOID
+EFIAPI
+IoMmuProtocolCallback (
+  IN  EFI_EVENT  Event,
+  IN  VOID       *Context
+  )
+{
+  EFI_STATUS  Status;
+
+  Status = gBS->LocateProtocol (&gEdkiiIoMmuProtocolGuid, NULL, (VOID **)&mIoMmuProtocol);

```

Comment:
Is there ever a case where multiple iommu protocols would be installed in a platform?